from pymongo import MongoClient
import datetime
# MongoDB connection string
uri = "mongodb+srv://######:#######!!@clusterasilvestre.zlkahyz.mongodb.net/?retryWrites=true&w=majority&appName=ClusteraSilvestre"

try:
    client = MongoClient(uri)
    db = client["ComuniArte"]

    usuarios = db["usuarios"]
    print("Conexi√≥n exitosa a MongoDB (Python)")


    #crear un usuario default:
    def crear_usuario(nombre, email, password, rol, activo):
        usuario = {
            "nombre": nombre,
            "email": email,
            "password": password,
            "rol": rol,
            "activo": activo,
            "creado_en": datetime.datetime.utcnow()
        }
        usuarios.insert_one(usuario)

except Exception as e:
    print("Error al conectar a MongoDB (Python):", e)
